.storybook/config.js

```diff
@@ -1,5 +1,9 @@
 import { configure, addDecorator } from '@kadira/storybook';
 import { setOptions } from '@kadira/storybook-addon-options';
+import _ from 'underscore';
+
+window._ = _;
```

Comment:
ah it's because of this https://github.com/NewSpring/Holtzmann/blob/9d13d35e5816d2e59a20007c6b61c365d69aef9f/imports/util/backgrounds/background.image.js#L11

good call!